web/src/components/alerting.tsx

```diff
@@ -613,10 +614,12 @@ const Tab: React.FC<{ active: boolean; children: React.ReactNode }> = ({ active,
 const AlertingPage: React.FC<RouteComponentProps<{ url: string }>> = ({ match }) => {
   const { t } = useTranslation(process.env.I18N_NAMESPACE);
   const { perspective } = usePerspective();
+  const { incidentsActive } = useFeatures();
 
   const alertsPath = getAlertsUrl(perspective);
   const rulesPath = getAlertRulesUrl(perspective);
   const silencesPath = getSilencesUrl(perspective);
+  const incidentsPath = '/monitoring/incidents';
```

Comment:
we should use a function that is aware of the perspective to get the link, for example in the case incidents needs to be enabled in ACM